frontend/public/components/utils/details-item.tsx

```diff
@@ -98,7 +109,9 @@ export const DetailsItem: React.FC<DetailsItemProps> = ({
             <>
               <SplitItem isFilled />
               <SplitItem>
-                <EditButton onClick={onEdit}>{t('public~Edit')}</EditButton>
+                <EditButton label={label} onClick={onEdit}>
```

Comment:
[nit] 'label' is kind of a misleading name for this prop. Maybe `testId` or something?